_data/versions.yml

```diff
@@ -1186,6 +1186,48 @@ v3.2:
       url: https://github.com/projectcalico/pod2daemon/releases/tag/v3.2.0
 
 v3.1:
+- title: v3.1.7
+  components:
+    calico/node:
+      version: v3.1.7
+      url: https://github.com/projectcalico/calico/releases/tag/v3.1.7
```

Comment:
This tag does not exist

Reply:
The release process for these versions has this tag pushed after this PR is merged.